butterknife/src/main/java/butterknife/internal/ButterKnifeProcessor.java

```diff
@@ -689,4 +690,18 @@ private void error(Element element, String message, Object... args) {
   private String getPackageName(TypeElement type) {
     return elementUtils.getPackageOf(type).getQualifiedName().toString();
   }
+
+  private static boolean hasAnnotationWithName(Element element, String simpleName) {
+    for (AnnotationMirror mirror : element.getAnnotationMirrors()) {
+      String annotationName = mirror.getAnnotationType().asElement().getSimpleName().toString();
+      if (simpleName.equals(annotationName)) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  private static boolean isRequiredInjection(Element element) {
+    return !hasAnnotationWithName(element, Nullable.class.getSimpleName());
```

Comment:
Just to be safe, and since I intend to delete our `@Nullable`, can we just put the string `"Nullable"` instead.